client/src/model/apis/navigationApi/navigationApi.js

```diff
@@ -10,7 +10,9 @@ export const createNavigationApi = ({ store }) => {
     createTraining: '/trainings/new',
     openTraining: '/trainings/:training',
     editTraining: '/trainings/:training/edit',
-    createExercise: '/trainings/:training/new-exercise',
+    editNewExercise: '/trainings/:training/editNew',
+    editExistingExercise: '/trainings/:training/:exercise/edit',
+    openExercise: '/trainings/:training/:exercise',
```

Comment:
right now I have a feeling that this flow `createExercise: '/trainings/:training/new-exercise'` should also exist. but let's discuss